specs/src/specs/params.md

```diff
@@ -0,0 +1,50 @@
+# Celestia Governance Params
+
+These are the parameters for mainnet. Note that not all of these parameters are
+changable via governance. This list also includes parameter that require a
+hardfork to change due to being manually hardcoded in the application or they
+are blocked by the `x/paramfilter` module.
+
+## Parameters
+
+| Module.Parameter | Default | Summary | Changeable via Governance |
+| --- | --- | --- | --- |
+| auth.MaxMemoCharacters | 256 | Largest allowed size for a memo in bytes. | True |
+| auth.TxSigLimit | 7 | Max number of signatures allowed in a multisig transaction. | True |
+| auth.TxSizeCostPerByte | 10 | Gas used per transaction byte. | True |
+| auth.SigVerifyCostED25519 | 590 | Gas used to verify Ed25519 signature. | True |
+| auth.SigVerifyCostSecp256k1 | 1000 | Gas used to verify secp256k1 signature. | True |
+| bank.SendEnabled | true | Allow transfers. | False |
+| blob.GasPerBlobByte | 8 | Gas used per blob byte. | True |
+| MaxBlockBytes | 100MiB | Hardcoded value in CometBFT for the protobuf encoded block. | False |
+| MaxSquareSize | 128 | Hardcoded value in the applications that requires a hardfork to change. | False |
+| blob.GovMaxSquareSize | 64 | Governance parameter for the  square size. If larger than MaxSquareSize, MaxSquareSize is used. | True |
+| consensus.block.MaxBytes | 1.88MiB | Governance parameter for the maximum size of the block. | True |
+| consensus.block.MaxGas | -1 | Maximum gas allowed per block (-1 is infinite). | True |
+| consensus.block.TimeIotaMs | 1000 | Minimum time added to the time in the header each block. | False |
+| consensus.evidence.MaxAgeNumBlocks | 100000 | The maximum number of blocks before evidence is considered invalid. This value will stop CometBFT from pruning block data. | True |
+| consensus.evidence.MaxAgeDuration | 2 days | The maximum age of evidence before it is considered invalid. This value should be identical to the unbonding period. | True |
+| consensus.evidence.MaxBytes | 1MB | Maximum size in bytes used by evidence in a given block. | True |
+| consensus.validator.PubKeyTypes | Ed25519 | The type of public key used by validators. | False |
+| consensus.Version.AppVersion | 1 | Determines protocol rules used for a given height. Incremented by the application upon an upgrade. | False |
+| distribution.communitytax | 0.02 | Percentage of the inflation sent to the community pool. | True |
+| distribution.WithdrawAddrEnabled | true | Enables delegators to withdraw funds to a different address. | True |
+| distribution.BaseProposerReward | 0 | Reward for proposing a block. | True |
+| distribution.BonusProposerReward | 0 | Extra reward for proposers based on the voting power included in the commit. | True |
+| gov.DepositParams.MinDeposit | 10000000utia | Minimum deposit for a proposal to enter voting period in seconds. | True |
```

Comment:
This is very low compared to other chains IMO. We should increase this! IMO at least 100TIA better 1000TIA. Mainly to prevent spam proposals. cc @YazzyYaz @musalbas 
Can be discussed in a separate issue though.

Reply:
Agree, also agree we can discuss it separately. First document current behavior.

Reply:
this is being updated in https://github.com/celestiaorg/celestia-app/pull/2417 and has been updated here, is okay to keep at 1000?

Reply:
I think it's ok at 1000 for now.